deploy/crds/push_v1alpha1_unifiedpushserver_cr_with_backup.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: push.aerogear.org/v1alpha1
```

Comment:
Hi @grdryn why you duplicated the  UnifiedPushServer instead of just create a new CR/CRD for the bkp? In this way, who decide to have the bkp just add the bkp CR. It is simple than the duplication.

Reply:
I didn't duplicate the UnifiedPushServer, I just provided an additional example CR. Now there's an example with backups in addition to the pre-existing example without backups.

A user _can_ add a backup (or remove or update one) without re-creating anything, since the backups get reconciled properly now (see the most recent commit).

Reply:
If I have the UPS installed and I would like to have the bkp, should I not remove the CR(service) then re-apply it with the bkp details as the new example? If yes, it does not means that the service will be deleted and re-created?